website/content/docs/drivers/index.mdx

```diff
@@ -10,11 +10,10 @@ Task drivers are used by Nomad clients to execute a task and provide resource
 isolation. By having extensible task drivers, Nomad has the flexibility to
 support a broad set of workloads across all major operating systems.
 
-Starting with Nomad 0.9, task drivers are now pluggable. This gives users the
-flexibility to introduce their own drivers without having to recompile Nomad.
-You can view the [plugin block][plugin] documentation for examples on how to
-use the `plugin` block in Nomad's client configuration. Note that we have
-introduced new syntax when specifying driver options in the client configuration
+Task drivers are pluggable. This gives users the flexibility to introduce their
+own drivers without having to recompile Nomad. You can view the [plugin block][plugin]
+documentation for examples on how to use the `plugin` block in Nomad's client configuration.
+Note that we have introduced new syntax when specifying driver options in the client configuration
```

Comment:
"we have introduced new syntax" is also version specific to 0.9. We should clean this sentence up too.

Reply:
Cleaned up and clarified in the latest commit